html/views/overlay/index.js

```diff
@@ -102,7 +102,7 @@ function initialize() {
 
 	WS.Register([ 'ScoreBoard.Settings.Setting(Overlay.Interactive.BackgroundColor)' ], function(k,v) {
 		if (k == 'ScoreBoard.Settings.Setting(Overlay.Interactive.BackgroundColor)') {
```

Comment:
Isn't this ìf` superfluous? We're registering on only this key. (And if it isn't, a comment might be warranted.)

Reply:
Fixed